IntelliLang.xml

```diff
@@ -0,0 +1,9 @@
+<application>
+  <component name="LanguageInjectionConfiguration">
```

Comment:
Let me google it for you: http://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste